etc/st2packgen/templates/action_template.yaml.jinja

```diff
@@ -21,11 +21,11 @@ parameters:
   {%- for parameter in paramsreq %}
   {{ parameter.name }}:
     type: {{ parameter.type }}
-    description: "{{ parameter.description }}"
```

Comment:
I'd done this because it solved errors related to unescaped characters. However, reverting the change, I don't get the errors anymore. \*sigh\*